### PR TITLE
Prelude: missing day_of_week

### PR DESCRIPTION
**Pull request by tranma**
! @jystic 
/jury approved @jystic

### CODE REVIEW

File: data/libs/prelude.icicle [hunk 1]
```diff
@@ -37,7 +40,7 @@ snd x = case x | (a,b) -> b end.
 group_keys k = keys (group k ~> 0).
 
 # This only works if inlining is set to with-subst (:set inline with-subst in the REPL)
-group_vals k v = vals (group k ~> v).
+# group_vals k v = vals (group k ~> v).
```
Comment (jystic) [inline data/libs/prelude.icicle]: why is this removed?

Comment (tranma) [inline data/libs/prelude.icicle]: see the comment, it's not always safe to use

Comment (jystic) [inline data/libs/prelude.icicle]: isn't that the default now? or not?

Comment (amosr) [inline data/libs/prelude.icicle]: I don't think so, I thought we decided to wait until the switch to Sorbet to enable that

Comment (jystic) [inline data/libs/prelude.icicle]: ah righto, perhaps we should remove it altogether then, I'm not sure what we gain by having it in there commented out?

**Comment by jystic** [comment]
👍 happy either way tbh
